app/client/sagas/transferAccountSagas.ts

```diff
@@ -89,8 +101,13 @@ function* watchLoadTransferAccounts() {
   yield takeEvery(LOAD_TRANSFER_ACCOUNTS_REQUEST, loadTransferAccounts);
 }
 
+interface TransferAccountEditApiResult {
+  type: typeof EDIT_TRANSFER_ACCOUNT_REQUEST;
+  payload: any;
+}
+
```

Comment:
lets move this to the TransferAccount types in https://github.com/teamsempo/SempoBlockchain/pull/298